migration/20221011164525-insert-countries-to-db.js

```diff
@@ -0,0 +1,11 @@
+const { MigrationInterface, QueryRunner } = require("typeorm");
+
+module.exports = class insertCountriesToDb20221011164525 {
+
+    async up(queryRunner) {
+        await queryRunner.query(
+            `insert into \`country\` (\`id\`,\`name\`) values ('3fa85f64-5717-4562-b3fc-2c963f66afa7','country 1');`
```

Comment:
This is not what I meant.
You should find a SQL query contains all countries and inserting them in the DB.
Use [this for countries](https://github.com/hiiamrohit/Countries-States-Cities-database/blob/master/countries.sql) and [this for states](https://github.com/hiiamrohit/Countries-States-Cities-database/blob/master/states.sql)